boards/DCM/Src/App/states/App_SharedStates.c

```diff
@@ -1,5 +1,9 @@
 #include "states/App_SharedStates.h"
 
+// PM100DZ Inverter Definitions
```

Comment:
can we be a bit more specific with the comments? this currently doesn't describe a lot

Reply:
Removed and instead added a comment within `App_SharedStates_HandleClearInvFaultsCmd`  which I think is pretty self-explanatory: 

"// Write a 0 to the fault clear parameter to clear any active faults"